tf-modules/ingest/python-reference-task.tf

```diff
@@ -0,0 +1,32 @@
+resource "aws_lambda_function" "python_reference_task" {
```

Comment:
We're exposing this as a task that end users can actually integrate? I'm not quite sure I understand why. I know having the code for reference is good - but as written the Lambda itself isn't very useful outside of testing?

Reply:
Hmm, that's a fair point, this should be pulled into the /example directory.

Reply:
That said, we're also providing `fake_processing_task` and `hello_world_task`